src/middlewared/middlewared/pytest/unit/plugins/test_smart.py

```diff
@@ -38,45 +67,64 @@ def test__parse_smart_selftest_results__ataprint__1():
 
 
 def test__parse_smart_selftest_results__ataprint__2():
-    assert parse_smart_selftest_results(textwrap.dedent("""\
-        Num  Test_Description    Status                  Remaining  LifeTime(hours)  LBA_of_first_error
-        # 1  Offline             Self-test routine in progress 100%         0         -
-    """)) == [
+    data = json.loads("""\
+{
+    "ata_smart_self_test_log": {
+        "standard": {
+            "revision": 1,
+            "table": [
+                {
+                    "type": {
+                        "value": 1,
+                        "string": "Offline"
+                    },
+                    "status": {
+                        "value": 249,
+                        "string": "Self-test routine in progress",
+                        "remaining_percent": 100,
+                        "passed": true
+                    },
+                    "lifetime_hours": 0
+                }
+            ],
+            "error_count_total": 0,
+            "error_count_outdated": 0
+            }
+        }
+}
+        """)
+    assert parse_smart_selftest_results(data) == [
         {
-            "num": 1,
+            "num": 0,
             "description": "Offline",
             "status": "RUNNING",
             "status_verbose": "Self-test routine in progress",
-            "remaining": 1.0,
+            "remaining": 100,
```

Comment:
Let's not change this please unless absolutely necessary

Reply:
I swear there was some valid reason I did this, I'll re-evaluate it again lol